lxd/db/cluster/update.go

```diff
@@ -106,6 +106,45 @@ var updates = map[int]schema.Update{
 	67: updateFromV66,
 	68: updateFromV67,
 	69: updateFromV68,
+	70: updateFromV69,
+}
+
+func updateFromV69(ctx context.Context, tx *sql.Tx) error {
+	_, err := tx.Exec(`
+CREATE TABLE identities (
+    id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
+    auth_method INTEGER NOT NULL,
```

Comment:
My thinking here is:
1. Having a `UNIQUE (type, identifier)` constraint is true, but not strict enough. It would mean that it would be possible for a client and a server certificate to have the same fingerprint.
2. It will be useful to have this index for calls to e.g. `GET /1.0/auth/identities/{authenticationMethod}`.

Reply:
Right, I see, thanks for clarifying, makes sense.